release.config.js

```diff
@@ -13,6 +13,25 @@ module.exports = {
     '@semantic-release/commit-analyzer',
     '@semantic-release/release-notes-generator',
     '@semantic-release/changelog',
+    [
+      '@semantic-release/exec',
+      {
+        // eslint-disable-next-line no-template-curly-in-string
+        prepareCmd: './prepareRelease.sh ${nextRelease.version}',
+      },
+    ],
+    [
+      '@semantic-release/npm',
+      {
+        npmPublish: false,
+      },
+    ],
+    [
+      '@semantic-release/git',
+      {
+        assets: ['package.json', 'src/main.ts'],
+      },
+    ],
```

Comment:
💡 The idea here is - 
- `prepareCmd` will exec will trigger our script with the nextRelease version.
- @semantic-release/npm will update the package.json with apt version. Since we don't want to publish to npm we set it to false
- @semantic-release/git will help us commit the changes in assets file list